cdap-ui/app/cdap/components/Replicator/Detail/TableScatterPlotGraph/ScatterPlotTable.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import { ITableMetricsData } from 'components/Replicator/Detail/TableScatterPlotGraph/parser';
+import numeral from 'numeral';
+
+const styles = (): StyleRules => {
+  return {
+    grid: {
+      '&.grid-wrapper': {
+        height: '100%',
+
+        '& .grid.grid-container.grid-compact': {
+          maxHeight: '415px',
+
+          '& .grid-header > .grid-row': {
+            alignItems: 'end',
+          },
+
+          '& .grid-row': {
+            gridTemplateColumns: '3fr 2fr 2fr 2fr 2fr',
+
+            '& > div:not(:first-child)': {
+              textAlign: 'right',
+            },
+          },
+        },
+      },
+    },
+  };
+};
+
+interface IScatterPlotTableProps extends WithStyles<typeof styles> {
+  data: ITableMetricsData[];
+}
+
+const ScatterPlotTableView: React.FC<IScatterPlotTableProps> = ({ classes, data }) => {
+  return (
+    <div className={`grid-wrapper ${classes.grid}`}>
+      <div className="grid grid-container grid-compact">
+        <div className="grid-header">
+          <div className="grid-row">
+            <div>Table name</div>
+            <div>
+              <div>Throughput</div>
+              <div>(events/min)</div>
+            </div>
+            <div>
+              <div>Avg. latency</div>
+              <div>min</div>
+            </div>
+            <div>
+              <div>Events</div>
+              <div>processed</div>
+            </div>
+            <div>Errors</div>
+          </div>
+        </div>
+
+        <div className="grid-body">
+          {data.map((table) => {
+            const eventsPerMin = numberFormat(table.eventsPerMin);
+            const latency = numberFormat(table.latency);
+
+            return (
+              <div className="grid-row" key={table.tableName}>
+                <div>{table.tableName}</div>
+                <div>{eventsPerMin}</div>
+                <div>{latency}</div>
+                <div>{table.totalEvents}</div>
+                <div>{table.errors}</div>
+              </div>
+            );
+          })}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+function numberFormat(d) {
+  return numeral(d).format('0.00a');
```

Comment:
Should we just move this to the helper function (`humanReadableNumber`). Right now we have a specific function for bytes, may be we could introduce one for decimal with units?

Reply:
moved to a helper function